sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 1999-2021 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.csp.sentinel.dashboard.handler;
+
+import com.alibaba.csp.sentinel.dashboard.client.CommandNotFoundException;
+import com.alibaba.csp.sentinel.dashboard.domain.Result;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.CollectionUtils;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.ServletWebRequest;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+
+/**
+ * @author chenglu
+ * @date 2021-06-02
+ */
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+    
+    private final Logger logger = LoggerFactory.getLogger(GlobalExceptionHandler.class);
+    
+    /**
+     * handle the {@link ExecutionException}, if the Exception's cause is {@link CommandNotFoundException}, then return code 4041.
+     *
+     * @param e {@link ExecutionException}
+     * @param webRequest webRequest
+     * @return fail message.
+     */
+    @ExceptionHandler({ExecutionException.class})
+    public Result<String> handleExecutionException(ExecutionException e, WebRequest webRequest) {
+        Map<String, String[]> params = webRequest.getParameterMap();
+        String contextPath = getRequestPath(webRequest);
+        logger.error("[HandleExecutionException] Request failed, context path: {}, request params: {}, error: ",
+                contextPath, buildParamMessage(params), e);
```

Comment:
Maybe `e.getCause()` is better here (for ExecutionException)?

Reply:
Hi @sczyh30 , I tried this. print the e will also print the cause exception. I think maybe we can keep it as now.